dispatcher/hub/commonHub.go

```diff
@@ -22,20 +21,14 @@ type ArgsCommonHub struct {
 }
 
 type commonHub struct {
-	filter                        filters.EventFilter
-	subscriptionMapper            dispatcher.SubscriptionMapperHandler
-	mutDispatchers                sync.RWMutex
-	dispatchers                   map[uuid.UUID]dispatcher.EventDispatcher
-	register                      chan dispatcher.EventDispatcher
-	unregister                    chan dispatcher.EventDispatcher
-	broadcast                     chan data.BlockEvents
-	broadcastRevert               chan data.RevertBlock
-	broadcastFinalized            chan data.FinalizedBlock
-	broadcastTxs                  chan data.BlockTxs
-	broadcastBlockEventsWithOrder chan data.BlockEventsWithOrder
-	broadcastScrs                 chan data.BlockScrs
-	closeChan                     chan struct{}
-	cancelFunc                    func()
+	filter             filters.EventFilter
+	subscriptionMapper dispatcher.SubscriptionMapperHandler
+	mutDispatchers     sync.RWMutex
+	dispatchers        map[uuid.UUID]dispatcher.EventDispatcher
+	register           chan dispatcher.EventDispatcher
```

Comment:
unused `register` and `unregister` channels?

Reply:
removed unused code